src/devices_models/devices/electric_loads.jl

```diff
@@ -97,6 +97,7 @@ function make_active_power_constraints_inputs(
                 variable_name = ACTIVE_POWER,
                 limits_func = x -> (min = 0.0, max = PSY.get_activepower(x)),
                 constraint_func = device_range,
+                constraint_struct = DeviceRangeConstraintInfo,
```

Comment:
These changes are also unrelated to this PR. Seems that you need to rebase or merge.

Reply:
The user needs to choose which constraint struct will be built by `device_range_constraints!`.